### PR TITLE
Rename GitHub Action names - App Services

### PR DESCRIPTION
**Pull request by vincent1173**

### CODE REVIEW

File: web-app-container/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM microsoft/azure-cli:2.0.47
 
 LABEL version="1.0.0"
 LABEL maintainer="Microsoft Corporation"
-LABEL com.github.actions.name="GitHub Action for Function App on Container"
+LABEL com.github.actions.name="Deploy to Azure WebappContainer"
```
Comment (SumiranAgg) [inline web-app-container/Dockerfile]: This is not similar to function container action name.

File: web-app-deploy/Dockerfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ FROM microsoft/azure-cli:2.0.47
 
 LABEL version="1.0.0"
 LABEL maintainer="Microsoft Corporation"
-LABEL com.github.actions.name="GitHub Action for Azure Web Apps"
+LABEL com.github.actions.name="Deploy to Azure WebApp"
```
Comment (SumiranAgg) [inline web-app-deploy/Dockerfile]: Should there be space between web and app?

**Review by SumiranAgg — APPROVED** [review]

File: web-app-container/Dockerfile (hunk 1 above)
Comment (vincentdass) [reply web-app-container/Dockerfile]: yes, the names are provided by PM.

File: web-app-deploy/Dockerfile (hunk 2 above)
Comment (vincentdass) [reply web-app-deploy/Dockerfile]: added. thanks

**Pull request merged**